TeachingRecordSystem/tests/TeachingRecordSystem.Api.Tests/V3/GetTeacherTestBase.cs

```diff
@@ -637,12 +655,18 @@ protected GetTeacherTestBase(ApiFixture apiFixture) : base(apiFixture)
                 It.IsAny<string[]>()))
             .ReturnsAsync(qtsRegistrations ?? Array.Empty<dfeta_qtsregistration>());
 
-        DataverseAdapterMock
-            .Setup(mock => mock.GetSanctionsByContactIds(new[] { contact.Id }, /* liveOnly: */ true))
-            .ReturnsAsync(new Dictionary<Guid, string[]>()
+        foreach (var sanction in sanctions ?? Array.Empty<(string, DateOnly?)>())
+        {
+            var sanctionCode = await TestData.ReferenceDataCache.GetSanctionCodeByValue(sanction.SanctionCode);
+
+            await TestData.OrganizationService.CreateAsync(new dfeta_sanction()
```

Comment:
This is a temporary work-around until we move to the new query pattern for all the data here.